taxi/admin.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Car, Driver
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ["model"]
+    list_filter = ["manufacturer"]
```

Comment:
Please specify which data you would like to see in this table.

Reply:
Done.